content/en/agent/kubernetes/distributions.md

```diff
@@ -0,0 +1,303 @@
+---
+title: Kubernetes distributions
+kind: documentation
+further_reading:
+    - link: 'agent/kubernetes/log'
+      tag: 'Documentation'
+      text: 'Collect your application logs'
+    - link: '/agent/kubernetes/apm'
+      tag: 'Documentation'
+      text: 'Collect your application traces'
+    - link: '/agent/kubernetes/prometheus'
+      tag: 'Documentation'
+      text: 'Collect your Prometheus metrics'
+    - link: '/agent/kubernetes/integrations'
+      tag: 'Documentation'
+      text: 'Collect automatically your applications metrics and logs'
+    - link: '/agent/guide/autodiscovery-management'
+      tag: 'Documentation'
+      text: 'Limit data collection to a subset of containers only'
+    - link: '/agent/kubernetes/tag'
+      tag: 'Documentation'
+      text: 'Assign tags to all data emitted by a container'
+---
+
+## Overview
+
+This section aims to document specificites and to provide good base configuration for all major Kubernetes distributions.
+These configuration can then be customized to add any Datadog feature.
+
+More `values.yaml` examples can be found in our [Helm chart repository][1]
+More `DatadogAgent` examples can be found in our [Datadog Operator repository][2]
+
+## AWS Elastic Kubernetes Service (EKS)
+
+No specific configuration is required unless your nodes run Bottlerocket.
+
+With Bottlerocket, specifying the `containerd` socket path is required to run the `containerd` check:
```

Comment:
Cannot we detect the runtime automatically with the latest Helm chart?

Reply:
If we need this, let's rephrase to: 

If you are using AWS Bottlerocket OS on your nodes, add the following to enable container monitoring:

Reply:
Being a non-default path we don't detect it automatically